PR_REVIEW_GUIDELINES.md

```diff
@@ -0,0 +1,70 @@
+      > 2. Click the settings button on the top left (the one with your avatar)
+      > 3. Navigate to “Your Workspace” > Reimburse Receipts > Connect bank account
+      > 4. Verify that the bank account view opens without errors
+      >
+    - Take note of the distinction between testing _locally_ and _on staging_. Note: The staging site references the production version of the API.
+3. Make sure that the test includes both **success** and **fail** scenarios.
+    - Ex: if the issue was to prevent the user from saving a field when it is empty, besides testing that we should also test that they can save the field _if it is not empty_.
+
+# Testing the changes
+
+1. Make sure there are no console errors related to the PR. If you find console errors in a branch you’re reviewing, check if they also exist in `main`.
+    1. If they do, proceed as normal (report them in #expensify-open-source if you think they need to be fixed).
+    2. If the errors do not exist in `main`, report it to the PR author - they need to be fixed before the PR can be merged.
+2. Test the changes on **all platforms**. Follow this guide (needs link) to set up each platform.
+    - If you’re unable to boot a platform for any reason, ask for help in the #expensify-open-source Slack channel. Your issue might already have been addressed there, so be sure to search first.
+
+# Reviewing the code
+
+## Good code patterns to require
+
+1. Check that functions have comments when appropriate.
+    - If the function has params or returns something, use [JSDocs syntax]((https://github.com/Expensify/App/blob/main/STYLE.md#jsdocs)) to describe them.
+        - Indicate the param name(s), data type passed and / or returned, and purpose if not immediately obvious.
+    - Obvious functions (with no params / return value) should not have comments.
+    - **In short: _Add comments & docs, only when useful._**
+2. All copy / text shown to users in the product should be stored in the `src/languages/*` files for localization.
+3. Platform-specific files should follow the proper naming convention (see [Platform-Specific File Extensions](https://github.com/expensify/app#platform-specific-file-extensions))
+    - Example: `MyComponent/index.android.js`
+
+## Code patterns to check for improvement
+
+1. Platform-specific solutions should not be applied to more platforms than necessary
+    - E.g. solutions for Android or iOS should go in specific `index.android.js` or `index.ios.js` files, not in an `index.native.js` file.
+2. Were any new styles added? Are we following the guidelines in [`STYLING.md`](./STYLING.md)?
+
+## Code patterns to avoid
+
+1. Make sure any `setTimeout`s are absolutely necessary, not just a workaround for a situation that isn’t fully understood (e.g. a race condition).
+2. Ensure Onyx is used appropriately. ([docs](https://github.com/expensify/react-native-onyx#merging-data))
```

Comment:
NAB, "Ensure Onyx is used appropriately." doesn't really fit with "Code patterns to avoid". Maybe we should mention a specific wrong way to use Onyx here?

Reply:
A few things I think we can note here
- Using Merge vs Set (Link to here: https://github.com/expensify/react-native-onyx#should-i-use-merge-or-set-or-both)
- Using the hoc to subscribe to data (like making the prop an appropriate name)

Do you think those points are useful to add?

Reply:
Yes, I think it makes more sense to call our some specific thing vs. the way this information is organized. We are saying 

>**Avoid:** Ensure Onyx is used appropriately

Reply:
Which is something I'd expect to see in "Good code patterns to require"